docrawl/docrawl_client.py

```diff
@@ -61,20 +61,24 @@ def get_browser_screenshot(self) -> str:
         return self.kv_redis.get(key=self._kv_redis_key_screenshot)
 
     def is_browser_active(self):
+        # NOTE: Not sure anywhere, it only checks if the process exists in a process list, not if it's active
         # TODO: finish later
-        pid = self.get_browser_meta_data()['browser'].get('pid', 0)
-
-        return psutil.pid_exists(pid)
+        pid = self.get_browser_meta_data()['browser']['pid']
```

Comment:
['browser'].get("pid") would be better when it was there

Reply:
`self.get_browser_meta_data()['browser']['pid']` can evaluate to `None`, crashing `psutil.pid_exists()`